conference/orders.py

```diff
@@ -0,0 +1,47 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+from django.db.models import Max
+
+from assopy.models import Order
+
+
+ORDER_CODE_PREFIX = "O/"
+ORDER_CODE_TEMPLATE = "O/%(year_two_digits)s.%(sequential_id)s"
+
+
+def increment_order_code(code):
+    NUMBER_OF_DIGITS_WITH_PADDING = 4
+
+    prefix_with_year, number = code.split('.')
+    number = str(int(number) + 1).zfill(NUMBER_OF_DIGITS_WITH_PADDING)
+    return "{}.{}".format(prefix_with_year, number)
+
+
+def latest_order_code_for_year(year):
+    """
+    returns latest used order.code in a given year.
+    rtype – string or None
+    """
+    assert 2016 <= year <= 2020, year
```

Comment:
Why is the assertion needed?

Reply:
mostly to make sure we're passing correct argument, not something like `'18'` or `'2018'`  for `2018`, etc. 
We probably don't need it anymore (it's a copy paste of the similar feature in conference/invoicing.py and was useful when refactoring last year). Upper bound is far enough in the future, lower bound is for how far back invoicing would work  with new setup (for example current code would fail on 2015 because we don't provide invoice issuer data for that year)